windows_install_build_dependencies.bat

```diff
@@ -166,8 +166,10 @@ mklink /d %BAZEL_SHORT_PATH%\openvino %BAZEL_SHORT_PATH%\%openvino_dir%
 if !errorlevel! neq 0 exit /b !errorlevel!
 
 :: Replace path to openvino in ovms WORKSPACE file
-powershell -Command "(gc -Path WORKSPACE) -replace '%openvino_workspace%', '%openvino_new_workspace%' | Set-Content -Path WORKSPACE"
-if !errorlevel! neq 0 exit /b !errorlevel!
+if "!output_user_root!" neq "opt" (
+    powershell -Command "(gc -Path WORKSPACE) -replace '%openvino_workspace%', '%openvino_new_workspace%' | Set-Content -Path WORKSPACE"
```

Comment:
Isn't there a CMD version of that? I wonder if going through powershell is something we want in a batch script

Reply:
Not possible with CMD, some features and not supported to make it a one liner.